src/common/components/header/Header.tsx

```diff
@@ -0,0 +1,20 @@
+import {Image, Flex, Button, Heading, HStack} from "@chakra-ui/react";
+
+function Header() {
+    return (
+        <HStack bg="white" h="72px" >
```

Comment:
Instead of doing a px height can you use a percentage?

Reply:
Changed to 8%